datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py

```diff
@@ -167,8 +115,16 @@ def __init__(self, name, init_config, agentConfig, instances, counter_list):
     def check(self, instance):
         self.log.debug("PDHBaseCheck: check()")
         key = hash_mutable(instance)
+        refresh_counters = is_affirmative(instance.get('refresh_counters', True))
+
+        if refresh_counters:
```

Comment:
Why are we having this in the check method now?

Reply:
behind flag in case someone encounters perf issues